code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -35,14 +35,17 @@ proc/issyndicate(mob/living/M as mob)
 	if(possible_syndicates.len < 1)
 		return 0
 
-	if(LAZYLEN(possible_syndicates) > agents_possible)
-		agent_number = agents_possible
+
+
+	if(num_players() >= required_players && num_players() < required_players + 20)
```

Comment:
Best to just make this into a sum.
`agent_number = min(required_enemies + round((num_players() - required_players) / 20), 5)`
Can make the 5 into a define or such to make it less of a magic number. Or just a variable on the game mode